airflow/www/static/css/flash.css

```diff
@@ -1,33 +1,33 @@
 /**
-* Licensed to the Apache Software Foundation (ASF) under one
-* or more contributor license agreements. See the NOTICE file
-* distributed with this work for additional information
-* regarding copyright ownership. The ASF licenses this file
-* to you under the Apache License, Version 2.0 (the
-* "License"); you may not use this file except in compliance
-* with the License. You may obtain a copy of the License at
-*
-* http://www.apache.org/licenses/LICENSE-2.0
-*
-* Unless required by applicable law or agreed to in writing,
-* software distributed under the License is distributed on an
-* "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
-* KIND, either express or implied. See the License for the
-* specific language governing permissions and limitations
-* under the License.
-*/
+ * Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
? Unnecessary space?

Reply:
This is what the licenses look like in the other files.  This license was added automatically and I had no influence on its content.
https://github.com/apache/airflow/blob/master/airflow/www/static/js/base.js
https://github.com/apache/airflow/blob/master/airflow/www/static/js/connection_form.js
https://github.com/apache/airflow/blob/master/airflow/www/static/js/datetime-utils.js
https://github.com/apache/airflow/blob/master/airflow/www/static/js/gantt-chart-d3v2.js
https://github.com/apache/airflow/blob/master/airflow/www/static/js/graph.js

Definition of comment style is available here: 
https://github.com/apache/airflow/blob/2b1de0b82d992ce16fbb7fc5becd18e26c113392/.pre-commit-config.yaml#L66

Reply:
In that case, all good

Reply:
Yeah. It's how it should be for consistency :).